src/constants/ErrResponses.ts

```diff
@@ -6,3 +6,12 @@ export const USER_ALREADY_EXISTS =
   'The provided email is aready linked to an account. Please check the email you have entered.';
 export const GENERIC_INTERNAL_ERROR =
   'There was an internal error. Please try again later.';
+
+// Auth Error Responses
+export const NO_PERMITTED_ROLES =
+  'The user does not have the required roles for this action.';
+export const USER_NOT_AUTHENTICATED = 'The user is unauthenticated.';
+
+// Validation Error Responses
+export const INVALID_REQUEST_BODY =
+  'The provided request body does not match with the schema it is supposed to follow.';
```

Comment:
we should parameterize this so that we tell the caller what part doesn't match

Reply:
You mean like turn it into a function?